src/components/App/App.js

```diff
@@ -5,62 +5,61 @@ import MoviePage from '../MoviePage/MoviePage';
 import './App.css';
 // import movieData from '../../data/movieData';
 
-
 class App extends React.Component {
-  constructor()  {
-    super()
-      this.state = {
-        movies: [],
-        isLoading: true,
-        currentMovie: "",
-        test: true,
-        error: false
-      }
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      isLoading: true,
```

Comment:
Try to limit your properties in state to only the data you absolutely need. Could render the loading component based on any other information in state to remove the need for `isLoading`?